perception/bytetrack/README.md

```diff
@@ -0,0 +1,97 @@
+| --------- | -------------------------------------------------- | ------------------------------------------- |
+| `in/rect` | `tier4_perception_msgs/DetectedObjectsWithFeature` | The detected objects with 2D bounding boxes |
+
+#### Output
+
+| Name                     | Type                                               | Description                                               |
+| ------------------------ | -------------------------------------------------- | --------------------------------------------------------- |
+| `out/objects`            | `tier4_perception_msgs/DetectedObjectsWithFeature` | The detected objects with 2D bounding boxes               |
+| `out/objects/debug/uuid` | `tier4_perception_msgs/DynamicObjectArray`         | The universally unique identifiers (UUID) for each object |
+
+### bytetrack_visualizer
+
+#### Input
+
+| Name       | Type                                                 | Description                                               |
+| ---------- | ---------------------------------------------------- | --------------------------------------------------------- |
+| `in/image` | `sensor_msgs/Image` or `sensor_msgs/CompressedImage` | The input image on which object detection is performed    |
+| `in/rect`  | `tier4_perception_msgs/DetectedObjectsWithFeature`   | The detected objects with 2D bounding boxes               |
+| `in/uuid`  | `tier4_perception_msgs/DynamicObjectArray`           | The universally unique identifiers (UUID) for each object |
+
+#### Output
+
+| Name        | Type                | Description                                                       |
+| ----------- | ------------------- | ----------------------------------------------------------------- |
+| `out/image` | `sensor_msgs/Image` | The image that detection bounding boxes and their UUIDs are drawn |
+
+## Parameters
+
+### bytetrack_node
+
+| Name                  | Type | Default Value | Description                                              |
+| --------------------- | ---- | ------------- | -------------------------------------------------------- |
+| `track_buffer_length` | int  | 30            | The frame count that a tracklet is considered to be lost |
+
+### bytetrack_visualizer
+
+| Name      | Type | Default Value | Description                                                                                   |
+| --------- | ---- | ------------- | --------------------------------------------------------------------------------------------- |
+| `use_raw` | bool | false         | The flag for the node to switch `sensor_msgs/Image` or `sensor_msgs/CompressedImage` as input |
+
+## Assumptions/Known limits
+
+## Reference repositories
+
+- <https://github.com/ifzhang/ByteTrack>
+
+## License
+
+The codes under the `lib` directory are copied from [the original codes](https://github.com/ifzhang/ByteTrack/tree/main/deploy/TensorRT/cpp) and modified.
```

Comment:
Could you change from main branch to hash?

Reply:
Thank you for the detailed review. I've changed the link to use a commit hash (b9fb69cc6a0e64481f43941870503ebc78ca57da).